src/debug_tools/diff.rs

```diff
@@ -0,0 +1,507 @@
+//! Diffing tools to compare two tries against each other. Useful when you want
+//! to find where the tries diverge from one other.
```

Comment:
nit: shouldn't it be `from one another`? (though my English skills are pretty limited..)

Reply:
Yep, typo!